src/clr-angular/layout/tabs/tab-content.ts

```diff
@@ -12,7 +12,14 @@ let nbTabContentComponents: number = 0;
 @Component({
   selector: 'clr-tab-content',
   template: `
+    <ng-template #tabContentProjectedRef>
+      <section [id]="tabContentId" role="tabpanel" [class.active]="active"
+               [attr.aria-labelledby]="ariaLabelledBy"
+               [attr.aria-hidden]="!active" [attr.data-hidden]="!active">
         <ng-content></ng-content>
+      </section>
+    </ng-template>
+    <ng-container [ngTemplateOutlet]="tabContentProjectedRef"></ng-container>
```

Comment:
I am adding a projection of the the template here, so the component can be instantiated and tested in isolation. Alternatively, I can change some tests that we have...

Reply:
Since you're using the template in the tabs component HTML, this is clearly not needed. I get that it simplifies tests, but it also runs the risk of instantiating too many contents by mistake in the future, impacting performance.

Reply:
Yes, it was leading to issues already when trying to support the no-clrIfActive case and I removed it from the latest code.